integration/shareddata/scalars.go

```diff
@@ -371,3 +373,27 @@ func tigrisSchema(typeString string) string {
 			}`
 	return strings.ReplaceAll(common, "%%type%%", typeString)
 }
+
+func generateBigMap(count int) map[string]any {
```

Comment:
Let's document this function (why and when it's needed)

Reply:
Added a comment.